casr/src/bin/casr-afl.rs

```diff
@@ -110,37 +126,40 @@ fn main() -> Result<()> {
             casr_tool: casr_gdb.clone(),
             ..Default::default()
         };
-        let cmdline_path = path.join("cmdline");
-        if let Ok(cmdline) = fs::read_to_string(&cmdline_path) {
-            crash_info.target_args = cmdline.split_whitespace().map(|s| s.to_string()).collect();
-            crash_info.at_index = crash_info
-                .target_args
-                .iter()
-                .skip(1)
-                .position(|s| s.contains("@@"))
-                .map(|x| x + 1);
+        crash_info.target_args = if matches.get_flag("ignore-cmdline") {
+            gdb_args.clone()
+        } else {
+            let cmdline_path = path.join("cmdline");
+            if let Ok(cmdline) = fs::read_to_string(&cmdline_path) {
+                cmdline.split_whitespace().map(|s| s.to_string()).collect()
+            } else {
+                error!("Couldn't read {}.", cmdline_path.display());
+                continue;
+            }
+        };
+        crash_info.at_index = crash_info
```

Comment:
Do we support stdin here?

Reply:
We will supply stdin if there is no `@@` in cmdline.